### DISCUSSION TITLE
Make Highlights more cutsomizable

### DISCUSSION THREAD

**Discussion by wuast94** [Chatterino/chatterino2#4710] [Ideas] [answered]
I have problems on the dark theme to see mentions, replys and overall highlightet messages.

They should be way more visible, even if looking to another screen.

Announcements are highlighted very strong but arent as importent in my opinion.

**Comment by Felanbird** [comment, marked as answer]
Change the color....?

![chatterino_2023-06-30_12-32-01](https://github.com/Chatterino/chatterino2/assets/41973452/8be3075e-864a-45fd-90f8-636f8270e65d)

**Reply by wuast94** [reply]
oh f**k, totally missed that, my bad
